.github/workflows/build_release_candidates.yml

```diff
@@ -37,3 +37,62 @@ jobs:
           name: client_linux_release_${{ github.sha }}
           path: build/dist
           if-no-files-found: error
+  android:
+    name: Build Client Android Release Candidate
+    runs-on: ubuntu-20.04
+    timeout-minutes: 15
+    environment: Client Android Release
+    env:
+      SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      ANDROID_KEY_STORE_PASSWORD: ${{ secrets.ANDROID_KEY_STORE_PASSWORD }}
```

Comment:
As discussed in https://github.com/Jigsaw-Code/outline-client/pull/1175, you can't use environments/deployments with external workflows. Keeping this inline for now.